util/app-config/src/tests/cli.rs

```diff
@@ -97,3 +98,55 @@ fn ba_advanced() {
 
     assert_eq!(1, sub_matches.occurrences_of(ARG_BA_ADVANCED));
 }
+
+#[test]
+#[ignore]
+/// the test needs bats installation and pre test environment setup, so skip this test
+/// in normal case(eg. `cargo test`)
+fn bats_suite() -> Result<(), std::io::Error> {
```

Comment:
Why use rust to execute the test suite? Why not just use Makefile or a bash file?

Reply:
most of tests in ckb are done by rust code(cargo test),  so i assume bats test follow this way.
yes, we could change into makefile/bash...

Reply:
It's unnecessary. Yes, just run bats from command line.

Reply:
now change mode to:  ci_yaml --> shell_do_real_job(cli_test.sh)--> call bats